tests/test_pre_commit_vauxoo.py

```diff
@@ -26,7 +26,7 @@ def setUp(self):
         super().setUp()
         self.old_environ = os.environ.copy()
         self.original_work_dir = os.getcwd()
-        self.tmp_dir = tempfile.mkdtemp(suffix="_pre_commit_vauxoo")
+        self.tmp_dir = os.path.realpath(tempfile.mkdtemp(suffix="_pre_commit_vauxoo"))
```

Comment:
Could you perform this on a separate commit, please?

Because it's not strictly related and to explain in commit message why it's necessary.